src/components/common/editor/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useMemo, useRef } from 'react';
+import ReactQuill from 'react-quill';
+import 'react-quill/dist/quill.snow.css';
+
+interface TextEditorProps {
+  onChange: (value: string) => void;
```

Comment:
event function이기 때문에 인자로 이벤트를 받아주셔야합니다!!
```suggestion
  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
```
Input 요소가 맞는지 모르겠지만,,, 그렇게 작성하심 됩니다